geemap/map_widgets.py

```diff
@@ -896,9 +900,14 @@ def _on_dropdown_click(self, change):
         if self.on_basemap_changed and change["new"]:
             self.on_basemap_changed(self._dropdown.value)
 
-    def _on_close_click(self, _):
+    def cleanup(self):
         if self.on_close:
             self.on_close()
+        if hasattr(self, 'toggle_off'):
+            self.toggle_off()
```

Comment:
Calling `toggle_off` seems like it would be error-prone (e.g. why doesn't inspector have it?). Is there a way we could do it automatically?